CHANGELOG.md

```diff
@@ -9,7 +9,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
-*
+* Reset value on `options` change in `select` component
```

Comment:
I know it is not related, but adding the issue that caused this is beneficial (perhaps add a comment in the Issue about the reasoning behind this change there)

Reply:
@beneditomauro Add the issue link here, atleast the one who originated this problem